modules/rostests/kmtests/kmtest_drv/kmtest_fsminifilter.c

```diff
@@ -514,12 +518,12 @@ FilterPreOperation(
 {
     FLT_PREOP_CALLBACK_STATUS Status;
     UCHAR MajorFunction;
-    INT i;
+    ULONG i;
 
     Status = FLT_PREOP_SUCCESS_NO_CALLBACK;
     MajorFunction = Data->Iopb->MajorFunction;
 
-    for (i = 0; i < sizeof(Callbacks) / sizeof(Callbacks[0]); i++)
```

Comment:
Did this refer to `static PFLT_OPERATION_REGISTRATION Callbacks = NULL;`, i.e. it was using `sizeof(PVOID) / sizeof(FLT_OPERATION_REGISTRATION)` before?

Reply:
Yes.

{{
c:\reactos-cov\modules\rostests\kmtests\kmtest_drv\kmtest_fsminifilter.c(522,18) :  warning: comparison of unsigned expression < 0 is always false [-Wtautological-unsigned-zero-compare]
    for (i = 0; i < sizeof(Callbacks) / sizeof(Callbacks[0]); i++)
                ~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
}}